examples/undecidable/README.md

```diff
@@ -0,0 +1,10 @@
+# Simple test example with 2 rooms
```

Comment:
Maybe we can add a brief description of the undecidability result, and cite arXiv.